### PR TITLE
Blog post about setting up Linux and Windows VMs in MacOS M1/M2

### PR DESCRIPTION
**Pull request by leo-bogastry**

### CODE REVIEW

**Review by PeuTit — APPROVED** [review]

**Comment by PeuTit** [comment]
Nice blog post. I'm just wondering if it would be possible to run an Intel based version of Windows?

**Comment by leo-bogastry** [comment]
> Nice blog post. I'm just wondering if it would be possible to run an Intel based version of Windows?

`If you were attempting
to run an Intel version of Windows 11 on an M1 or M2 Mac, you’d need to choose `Emulate` instead, which is expected to operate at a slower pace.`

I'm wondering if you read the blog or you just want pastel de nata 🤣

**Comment by PeuTit** [comment]
Ahah I did read the post. I miswrote my question, it was actually the opposite, could we emulate an OS (ARM) on an Intel MacBook Pro?

**Comment by leo-bogastry** [comment]
> Ahah I did read the post. I miswrote my question, it was actually the opposite, could we emulate an OS (ARM) on an Intel MacBook Pro?

Does this question still involves windows? 
I never had to test anything in two different types of macbook, I think for that you need the actual Apple silicon or Intel-based laptop. It's very much a hardware issue. There's Rosetta, but it's only available for Apple silicon laptops

**Comment by PeuTit** [comment]
I guess the question can be extended to any x86 OS. Thanks for the answer, I was just curious if you came across something like this while researching your issue.

File: posts/2023-10-23-run-linux-windows-on-mac.adoc [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+You should now be able to enjoy your Windows VM with an active internet connection.
+
+.Windows VM with internet
+image::utm-windows-with-internet.png[Windows with internet, 600]
+
+
+== Conclusion
+In this blog post, we successfully set up both a Linux and a Windows VM. The Linux VM was configured
+using XCode, the ‘RunningGUILinuxInAVirtualMachineOnAMac’ project, and a free Linux distribution, requiring minimal
```
Comment (sekipp) [inline posts/2023-10-23-run-linux-windows-on-mac.adoc:160]: My suggestion is to tweak this to: The Linux VM was configured using XCode, the ‘RunningGUILinuxInAVirtualMachineOnAMac’ project, and a free Linux distribution--all requiring minimal configuration.

**Review by PeuTit — APPROVED** [review]

**Pull request merged**